### PR TITLE
[Behavioural QC] Add subproject filter

### PR DESCRIPTION
**Pull request by CamilleBeau**
## Brief summary of changes
This PR adds a subproject drop-down in the filters for Behavioural QC. This change was made on CCNA but should be relevant / useful to other projects as well.

#### Testing instructions (if applicable)

1. Make sure that module loads properly
2. Make sure that all drop downs in the filters are populated with the correct options
3. Test the filters and make sure that the results are updated accordingly 
4. Make sure you can clear the filters, and the results are reset

### CODE REVIEW

File: modules/behavioural_qc/php/views/conflicts.class.inc [hunk 1]
```diff
@@ -27,15 +27,23 @@ class Conflicts
...
+        $subprojects,
+        $sites
+    ) {
         $this->_formElement = [
             'instruments' => $instruments,
             'visits'      => $visits,
             'projects'    => $projects,
+            'subprojects' => $subprojects,
             'sites'       => $sites,
+            'subprojects' => $subprojects,
```
Comment (HenriRabalais) [inline modules/behavioural_qc/php/views/conflicts.class.inc:46]: You seem to have added subprojects twice here.

**Review by HenriRabalais — APPROVED** [review]
One small typo but other than that LGTM!

**Review by HenriRabalais — CHANGES_REQUESTED** [review]
Meant to hit request changes!

File: modules/behavioural_qc/php/views/conflicts.class.inc (hunk 1 above)
Comment (CamilleBeau) [inline modules/behavioural_qc/php/views/conflicts.class.inc:46]:
```suggestion
            'subprojects' => $subprojects,
            'sites'       => $sites,
```

Comment (CamilleBeau) [reply modules/behavioural_qc/php/views/conflicts.class.inc]: Thanks!

**Review by HenriRabalais — APPROVED** [review]

**Pull request merged**